docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -0,0 +1,1270 @@
+        entry in the storage section which provides named lists of hosts. The
+        <emphasis>hostGroups</emphasis> entries can take one of two forms.
+        This is the most common form, and directly associates a list of host
+        names with a name:</para>
+
+        <programlisting>storage: 
+  hostGroups: 
+  - name:  &lt;name&gt; "The name of the host group" 
+    hosts: [ "a list of host names" ] 
+</programlisting>
+
+        <para>The second form allows one host group to be derived from
+        another:</para>
+
+        <programlisting>storage: 
+  hostGroups: 
+  - name: "The name of the host group process" 
+    hostGroup: "Name of the hostgroup to create a subset of" 
+    count: &lt;Number of hosts in the subset&gt; 
+    offset: &lt;the first host to include in the subset&gt;  
+    delta: &lt;Cycle offset to apply to the hosts&gt; 
+</programlisting>
+
+        <para>Some typical examples with bare-metal clusters are smaller
+        subsets of the host, or the same hosts, but storing different parts on
+        different nodes, for example:</para>
+
+        <programlisting>storage: 
+  hostGroups: 
+  - name: groupABCDE # Explicit list of hosts 
+    hosts: [A, B, C, D, E] 
+  - name groupCDE # Subset of the group last 3 hosts 
+    hostGroup: groupABCDE 
+    count: 3 
+    offset: 2  
+  - name groupDEC # Same set of hosts, but different part-&gt;host mapping 
+    hostGroup: groupCDE 
+    delta: 1</programlisting>
+
+        <para>The second aspect is to add a property to the storage plane
+        definition to indicate which hosts are associated with it. There are
+        two options:</para>
+
+        <itemizedlist>
+          <listitem>
+            <para><emphasis role="bold">hostGroup: &lt;name&gt;</emphasis> The
+            name of the host group for bare metal. The name of the hostGroup
+            must match the name of the storage plane.</para>
+          </listitem>
+
+          <listitem>
+            <para><emphasis role="bold">hosts:
+            &lt;list-of-namesname&gt;</emphasis> An inline list of hosts.
+            Primarily useful for defining one-off external landing
+            zones.</para>
+          </listitem>
+        </itemizedlist>
+
+        <para>For Example:</para>
+
+        <programlisting>storage: 
+  planes: 
+  - name: demoOne 
+    category: data 
+    prefix: "/home/demo/temp" 
+    hostGroup: groupABCD # The name of the hostGroup 
+  - name: myDropZone 
+    category: lz 
+    prefix: "/home/demo/mydropzone" 
+    hosts: [ 'mylandingzone.com' ] # Inline reference to an external host. </programlisting>
+      </sect3>
+    </sect2>
+
+    <sect2 id="StorageItems_HPCC_Systems_Coomponents">
+      <title>Storage Items for HPCC Systems Components</title>
+
+      <sect3 id="YML-DOC_GenData-Storage">
+        <title>General Data Storage</title>
+
+        <para>General data files generated by HPCC are stored stored in data.
```

Comment:
should be a \<sect4> for data category for consistency

Reply:
this one is tricky because it would require adding some content at the sect3 level to be able to move this down to the sect4 level... instead I am going to bump the subsequent sect4 headings up to the sect3 to make this section more uniform.